src/modes/thread_view/thread_view.cc

```diff
@@ -1385,6 +1385,20 @@ namespace Astroid {
           return true;
         });
 
+    keys.register_key (Key (GDK_KEY_BackSpace),
```

Comment:
Why is it sometimes Backspace and sometimes Delete? Should always be the same I think. I think that we can use `d`

Reply:
I will look again but I believe `d` is already taken in the default keybindings for one of the modes.